library/CMakeLists.txt

```diff
@@ -92,7 +92,7 @@ if(CMAKE_COMPILER_IS_CLANG)
 endif(CMAKE_COMPILER_IS_CLANG)
 
 if(WIN32)
-    set(libs ${libs} ws2_32)
+    set(libs ${libs} ws2_32 bcrypt)
```

Comment:
This prevents building for old Windows versions, not just with Visual Studio but with e.g. mingw. Do we want that?

Not necessarily a blocker anyway, we can tell people with special needs (e.g. Windows XP) to edit the build scripts.

Reply:
That's a decision we've already made. Please see the new [policy](https://tls.mbed.org/tech-updates/blog/retiring-legacy-windows-support) on it.

Basically, this is going into `devlopment` and the older LTS branches will be maintained for older Windows installations.

Reply:
@gilles-peskine-arm I do maintain patchsets for recent versions of mbed TLS, as I tend to support Windows 2000/XP as a side effect of linking against the system libc `msvcrt.dll`. (Of course, depending on the rest of the project, this is not always guaranteed.)

(Yes, this has been discouraged since 1999. This is why import libraries for the recent versioned libc are included with mingw-w64. But GCC continues doing so by default because it assumes a single, systemwide `libc.so` as on UNIX and Linux)